docs/operating/extending.md

```diff
@@ -70,38 +70,18 @@ See examples of this in [/examples/extensions/main.go](/examples/extensions/main
 Middleware enables you to add functionality to every API request. For every request, the chain of Middleware will be called 
 in order allowing you to modify or reject requests, as well as write output and cancel the chain. 
 
-NOTES:
-
-* middleware is responsible for writing output if it's going to cancel the chain.
-* cancel the chain by returning an error from your Middleware's Serve method.
-
-## Special Handlers
-
-To understand how **Special Handlers** works you need to understand what are **Special Routes**.
-
-**Special Routes** are routes that doesn't match any other API route. 
-
-With **Special Handlers** you can change the behavior of special routes in order to define which function is going to be executed.
-
-For example, let's use special handlers to define `mydomain` as the `appname` for any request for `mydomain.com`.
+There are two separate sets of middleware, one for the IronFunctions API and one that wraps requests to user created endpoints. 
```

Comment:
A little misleading? As only the API endpoints are wrapped by middleware not the functions invokers: https://github.com/iron-io/functions/blob/ce26f665ea2b1e83ce093ba0f9fe86d27b055e11/api/server/server.go#L252-L271

Or is it supposed to encapsulate the functions endpoints too?